app/views/navbar.scala.html

```diff
@@ -71,49 +71,49 @@
                 }
                 @if(url.isDefined && url.get == "/audit") {
                     <li class="active navbar-lnk">
-                        <a class="navbar-button" href="#" id="toolbar-onboarding-link">@Messages("navbar.retake.tutorial")</a>
+                        <a class="navbar-button" href="#" id="navbar-retake-tutorial-btn">@Messages("navbar.retake.tutorial")</a>
                     </li>
                 }
                 @if(url.isDefined && url.get != "/audit") {
                     <li class="active navbar-lnk">
-                        <a class="navbar-button navbar-start-btn" href="@routes.AuditController.audit()">@Messages("navbar.explore")</a>
+                        <a class="navbar-button" id="navbar-start-btn" href="@routes.AuditController.audit()">@Messages("navbar.explore")</a>
                     </li>
                 }
                 @if(url.isDefined && url.get != "/validate") {
                     <li class="active navbar-lnk">
-                        <a class="navbar-button navbar-validate-btn" href="@routes.ValidationController.validate()">@Messages("navbar.validate")</a>
+                        <a class="navbar-button" id="navbar-validate-btn" href="@routes.ValidationController.validate()">@Messages("navbar.validate")</a>
                     </li>
                 }
                 <li class="active navbar-lnk">
-                    <a class="navbar-button navbar-guide-btn" href="@routes.ApplicationController.labelingGuide">@Messages("navbar.howto")</a>
+                    <a class="navbar-button" id="navbar-guide-btn" href="@routes.ApplicationController.labelingGuide">@Messages("navbar.howto")</a>
                 </li>
                 @if(url.isDefined && url.get == "/audit") {
                     <li class = "active navbar-lnk">
-                        <a class="navbar-button" href='@routes.ApplicationController.help' id="toolbar-help-link" target="_blank">@Messages("navbar.help")</a>
+                        <a class="navbar-button" id="navbar-help-btn" href='@routes.ApplicationController.help' target="_blank">@Messages("navbar.help")</a>
                     </li>
                 }
                 <li class="active dropdown navbar-lnk" id="navbar-data-dropdown-list">
-                    <a id="nav-user-dropdown" class="navbar-button" role="button" data-toggle="dropdown" href="#">
+                    <a id="nav-data-dropdown" class="navbar-button" role="button" data-toggle="dropdown" href="#">
                         @Messages("navbar.data")
                         <b class="caret"></b>
                     </a>
-                    <ul id="nav-user-menu" class="dropdown-menu" role="menu" aria-labelledby="User menu">
+                    <ul id="nav-data-menu" class="dropdown-menu" role="menu" aria-labelledby="User menu">
```

Comment:
I assume that "User menu" should be "Data menu"

Reply:
Actually, after taking a look at [some documentation](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-labelledby_attribute), it looks like this field was being used incorrectly already. It looks like this should maybe just be `aria-labelledby="navbar-data-dropdown-list"`? Or maybe we just don't even need that attribute? Can you investigate a bit and fix it up for both drop down menus? It may seem small, but I'd like our navbar to be read by screen readers correctly.